### PR TITLE
Update nut.Markdown

### PR DESCRIPTION
**Pull request by sparkydave1981**
Include detail for connecting to the NUT add-on using the slug as the host address

## Proposed change
Include details for connecting to the NUT add-on using the slug as the host address as using 'localhost' or the HA server's IP does not work.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

This is my first attempt at a PR for the docs so please go easy on me!

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @sparkydave1981, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#22414).

Once you do that we will be able to review and accept this pull request.

Thanks!

**Comment by homeassistant** [comment]
Hi @sparkydave1981, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#22414).

Once you do that we will be able to review and accept this pull request.

Thanks!

File: source/_integrations/nut.markdown [hunk 1]
```diff
@@ -23,6 +23,11 @@ server.
 
 {% include integrations/config_flow.md %}
 
+## Connecting to the NUT Add-on
+
+To connect to the NUT add-on the slug of the add-on is required as the host address, in this case a0d7b954-nut. You can then use the default port number (unless changed) and the username & password you entered under the add-on configuration.
```
Comment (frenck) [inline source/_integrations/nut.markdown:28]: This should not be part of the Home Assistant documentation, as it is specific for a use case with a third-party add-on. Additionally, there are multiple addresses (and thus the information is incomplete).

This comment/instruction fits better into the add-on documentation of that specific add-on.

**Comment by frenck** [comment]
Closing this PR as per the above comment.

Thanks for willing to contribute @sparkydave1981 👍 

../Frenck

**Pull request closed without merging**